release/test_release_gold.sh

```diff
@@ -52,4 +54,24 @@ function test_invoke_github_api_post {
 	assert_equals $(invoke_github_api_post "liferay-portal-ee/git/tags" "${tag_data}") "${LIFERAY_COMMON_EXIT_CODE_OK}" $(invoke_github_api_post "liferay-portal-ee/git/refs" "${ref_data}") "${LIFERAY_COMMON_EXIT_CODE_OK}"
 }
 
+function test_update_release_info_date {
+	_PRODUCT_VERSION="2024.q2.11"
+
+	update_release_info_date --test
+
+	assert_equals "${?}" "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+
+	_PRODUCT_VERSION="2024.q2.12"
+
+	update_release_info_date --test
+
+	assert_equals "${?}" "${LIFERAY_COMMON_EXIT_CODE_OK}"
+
+	_PRODUCT_VERSION="2024.q3.0"
+
+	update_release_info_date --test
+
+	assert_equals "${?}" "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
```

Comment:
Almost there, @kiwm. :slightly_smiling_face: 

Can you please remove these `echos` from the test output?

![image](https://github.com/user-attachments/assets/ee5d9486-62c5-4851-a647-6235c76d5a70)

Additionally, can you please include tests for 7.4 U, 7.4 GA and 7.3?

Thanks in advance